app/css/_sidebar.scss

```diff
@@ -0,0 +1,71 @@
+.sidebar-right-aside{
+  min-width: $sidebar-min-width;
```

Comment:
It means you shall not declare any mobile related styles inside the breakpoints, but respecify styles for other "bigger" devices.